apps/data/src/lib/enrich/enrich/macro_execution.py

```
@@ -0,0 +1,377 @@
+"""
+Macro Execution via Backend API
+
+Provides a pandas UDF factory for executing macros through the openJII
+backend API (Lambda/Firecracker sandbox) instead of locally on Databricks.
+Used to create a side-by-side comparison table for validating the sandbox
+approach against the existing local macro execution.
+"""
+
+from typing import Any, Dict, List
```

Ruff (0.15.10)

[warning] `typing.Dict` is deprecated, use `dict` instead

(UP035)